### DISCUSSION TITLE
龙猫，求求了开个联机功能吧

### DISCUSSION THREAD

**Discussion by Huan-Starvm** [Meloong-Git/PCL#2008] [提问] [answered]
国内内网穿透服务器我可以提供，内蒙古的，求求开个联机功能吧（ 

----来自一个傻鸟的请求

**Comment by WForst-Breeze** [comment, marked as answer]
别急，请先等待 Octo 发布，请见 [2.6.2 更新日志](https://www.bilibili.com/read/cv23979077)：*关于联机，先等到 8 月看看……没法急。*

**Reply by Huan-Starvm** [reply]
缺少服务器内网穿透可以随时找我，我可以提供一台内蒙的30M的服务器（ 当作小小赞助下
